src/pyroe/make_txome.py

```diff
@@ -366,22 +366,22 @@ def check_bedtools_version(bt_check_path):
                 # in this case, there's nowhere else to check
                 # so give up on bedtools
                 print(
-                    "bedtools in the environemnt PATH is either",
+                    "bedtools in the environemnt PATH either is",
                     "older than v.2.30.0 or doesn't exist.",
                     "\nBiopython will be used.",
                 )
                 no_bt = True
             else:
                 print(
-                    "bedtools specified by bt_path is either",
```

Comment:
Actually, here I think the previous wording is more "idiomatic" in English.  The thing "is either A or B" is more common than "either is A or B".